python/src/TensorProductExperiment_doc.i.in

```diff
@@ -0,0 +1,173 @@
+%feature("docstring") OT::TensorProductExperiment
+"Tensor product experiment.
+
+Parameters
+----------
+experiments : list of :class:`~openturns.WeightedExperiment`
+    List of :math:`n` marginal experiments of the tensor product experiment. 
+    Each marginal experiment can have arbitrary dimension.
+
+Notes
+-----
+The tensor product design of experiments (DOE) is based on a collection 
+of marginal multidimensional elementary designs of experiments. 
+It is anisotropic in the sense that each marginal DOE does not necessarily 
+have the same size. Using more points in one component may be useful if we 
+want to get more accurate results in that particular direction. 
+
+Furthermore, each marginal DOE does not necessarily have a dimension 
+equal to 1. In this sense, this is a generalization of a classical 
+tensor product DOE. 
+
+Let :math:`\mathcal{X} \subset \mathbb{R}^{d_x}` be the integration domain 
+and let :math:`g : \mathcal{X} \rightarrow \mathbb{R}^{d_y}` be an integrable 
+function. 
+Let :math:`f : \mathcal{X} \rightarrow \mathbb{R}` be a probability density 
+function. The tensor product experiment produces an approximation of the 
+integral :
+
+.. math::
+
+    \int_{\mathcal{X}} g(\boldsymbol{x}) f(\boldsymbol{x}) d\boldsymbol{x} 
+    \approx \sum_{i = 1}^{s_t} w_i g\left(\boldsymbol{x}_i\right)
+
+where :math:`s_t \in \mathbb{N}` is the size of the tensor product 
+design of experiments, :math:`w_1, ..., w_{s_t} \in \mathbb{R}` are the 
+weights and :math:`\boldsymbol{x}_1, ..., \boldsymbol{x}_{s_t} \in \mathbb{R}^{d_x}` 
+are the nodes. 
+
+Let :math:`n_e \in \mathbb{N}` be the number of marginal DOEs. 
+For each marginal DOE index :math:`j = 1, ..., n_e`, 
+let :math:`s_j \in \mathbb{N}` be its size, representing the number of 
+nodes in the marginal DOE. 
+For any index :math:`j = 1, ..., n_e`, let :math:`d_j \in \mathbb{N}` be 
```

Comment:
```suggestion
nodes in the marginal DOE, and
let :math:`d_j \in \mathbb{N}` be 
```
With this change, the sentence is longer, but clearer IMO. What do you think @mbaudin47 ?

Reply:
Very true. Ok. Fixed.